diff --git a/node_modules/chartist/dist/chartist.js b/node_modules/chartist/dist/chartist.js
index 68536a7..0329a87 100644
--- a/node_modules/chartist/dist/chartist.js
+++ b/node_modules/chartist/dist/chartist.js
@@ -993,6 +993,9 @@ var Chartist = {
     positionalData[axis.counterUnits.pos] = labelOffset[axis.counterUnits.pos];
     positionalData[axis.units.len] = length;
     positionalData[axis.counterUnits.len] = Math.max(0, axisOffset - 10);
+    if (axis.counterUnits.len === 'width') {
+      positionalData[axis.counterUnits.len] = 300;
+    }
 
     if(useForeignObject) {
       // We need to set width and height explicitly to px as span will not expand with width and height being
@@ -2171,9 +2174,6 @@ var Chartist = {
       content = container.firstChild;
     }
 
-    // Adding namespace to content element
-    content.setAttribute('xmlns', Chartist.namespaces.xmlns);
-
     // Creating the foreignObject without required extension attribute (as described here
     // http://www.w3.org/TR/SVG/extend.html#ForeignObjectElement)
     var fnObj = this.elem('foreignObject', attributes, className, insertFirst);
